ui/yarn.lock

```diff
@@ -15361,6 +15361,11 @@ lodash@4.13.1:
   version "4.13.1"
   resolved "https://registry.yarnpkg.com/lodash/-/lodash-4.13.1.tgz#83e4b10913f48496d4d16fec4a560af2ee744b68"
 
+lodash@4.17.15, lodash@^4.0.1, lodash@^4.17.12, lodash@^4.17.13, lodash@^4.17.14, lodash@^4.17.15:
```

Comment:
@josephthweatt just FYSA, I've been merging a few dependabot PRs that upgrade lodash to 4.17.15 in a few PRs.

Reply:
I merged in all the PRs that upgraded lodash to 4.17.15 in their respective package.json files.

Reply:
lodash changes removed after rebasing